### PR TITLE
Update peer dep with Tailwindcss

### PR DESCRIPTION
**Pull request by aFuzzyBear**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=pEnoTw3)
## Changes
updated `package.json` with `tailwindcss` as a peer dependency of the integration itself. 

Unsure if `changeset` is needed

## Testing

N/A

## Docs
N/A

### CODE REVIEW

**Comment by FredKSchott** [comment]
**Is this a breaking change?** I think so. It *might* not be since tailwind is still a dependency, but in that case this change isn't actually doing anything. It makes more sense to remove `tailwind` as a "dependency", only list it as a "peerDependency", and call this a major release of the integration.

**Is this going to break existing users?** It shouldn't as long as we correctly changeset it as a major bump, with good information in the changelog! But it's definitely a big change and we should talk about timing before merging this PR.

**Comment by aFuzzyBear** [comment]
Thanks @FredKSchott, I made the amendment, in respects to the second question I would defer to your wisdom on this one.

File: .changeset/leaders-of-the-free-world.md
Comment (bluwy) [inline .changeset/leaders-of-the-free-world.md:2]:
```suggestion
"@astrojs/tailwind": major
```

I believe this should be a major.

**Review by bluwy — APPROVED** [review]

**Comment by matthewp** [comment]
What's the status of this one @aFuzzyBear, is it ready to go?

**Comment by aFuzzyBear** [comment]
> What's the status of this one @aFuzzyBear, is it ready to go?

pretty much yes I believe,  @bluwy did sign off on it, wasn't sure if there was anything else needed to be done,   I'm hesitatent to merge this myself, unless given the okay,

**Pull request merged**
